cmake/configure/configure_3_petsc.cmake

```diff
@@ -103,6 +103,15 @@ MACRO(FEATURE_PETSC_CONFIGURE_EXTERNAL)
   SET(DEAL_II_EXPAND_PETSC_BLOCKVECTOR "PETScWrappers::BlockVector")
   SET(DEAL_II_EXPAND_PETSC_MPI_VECTOR "PETScWrappers::MPI::Vector")
   SET(DEAL_II_EXPAND_PETSC_MPI_BLOCKVECTOR "PETScWrappers::MPI::BlockVector")
+  #
+  # FIXME:
+  # temporary variable until deal.II fully support complex-valued PETSc
+  IF(NOT PETSC_USE_COMPLEX)
```

Comment:
don't understand it, `petscconf.h` has 
```
#ifndef PETSC_USE_COMPLEX
#define PETSC_USE_COMPLEX 1
#endif
```
but still i always get inside this `IF`...